.eslintrc

```diff
@@ -0,0 +1,46 @@
+{
+  "root": true,
+  "extends": ["eslint:recommended", "google"],
+  "plugins": [
+    "jsdoc"
+  ],
+  "parserOptions": {
+    "ecmaVersion": 2018
```

Comment:
I'd imagine this can be `6`, since we're using ES6 code?

Reply:
Since this is going to run on node > 10, might as well take advantage of the latest language features :)